great-idea-website/css/index.css

```diff
@@ -63,4 +63,92 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	width: 878px;
+	margin: auto;
+}
+/* header */
+	header nav {
+		display: inline-block;
+		margin-top: 35px;
+		margin-left: 40px;
+	}
+
+	header nav a {
+		margin: 0 20px;
+		color: gray;
+		text-decoration: none;
+	}
+
+	header img.logo {
+		margin-left: 50px;
+	}
+/* /header */
+
+/* cta */
+	.cta-text {
+		display: inline-block;
+		font-size: 70px;
+		text-align: center;
+		vertical-align: top;
+		padding: 55px;
+		margin-left: 25px;
+		margin-top: 30px;
+	}
+	.cta-text button {
+		font-size: 20px;
+		background-color: white;
+	}
+	.circle-code {
+		display: inline-block;
+		vertical-align: top;
+		margin-left: 60px;
+		margin-top: 45px;
+	}
+/* cta */
+
+/* main-content */
+	section div.top-content {
+		width: 100%;
+		display: inline-block;
+	}
+	div.top-content div.text-content {
+		display: inline-block;
+		vertical-align: top;
+		width: 49%;
+		margin: 30px 0;
+	}
+
+	div.bottom-content{
+		display: inline-block;
+		margin-top: 30px;
+		margin-bottom: 30px;
+	}
+	div.bottom-content div.text-content-0 {
+		display: inline-block;
+		width: 32%;
+		margin-right: 1%;
+	}
+	div.bottom-content div.text-content-1 {
+		display: inline-block;
+		width: 32%;
+		margin-right: 1%;
+	}
+	div.bottom-content div.text-content-2 {
+		display: inline-block;
+		width: 32%;;
+	}
```

Comment:
The rules for the three bottom `text-content` is nearly the same, so another way you can do this is to just have one with all three rules, and then another block which removes the `margin-right` from `text-content-2`.